packages/discovery-react-components/src/components/DiscoverySearch/DiscoverySearch.tsx

```diff
@@ -246,7 +249,7 @@ export const DiscoverySearch: FC<DiscoverySearchProps> = ({
         }
       });
     },
-    [performSearch, setSearchParameters]
+    [performSearch, searchClient, setSearchParameters]
```

Comment:
I've removed this previously but I believe it keeps getting added in when we autofix with eslint before commit